scaffold/github/workflows/ComposerLockDiffComment.yml

```diff
@@ -0,0 +1,12 @@
+name: "Post Comment on a Github issue/PR with the composer-lock-diff contents"
+
+on:
+  push
+
+jobs:
+  Post-Comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Posting a single comment on this PR
```

Comment:
I'm not sure about the name "Posting a single comment on this PR" for this step will help us in the future debug thing.

... I wonder where that shows up... seems like here, inside the action

<img width="1813" alt="Screen Shot 2022-09-26 at 5 26 33 PM" src="https://user-images.githubusercontent.com/1970586/192383762-b4610259-d319-4120-947f-58fedd299ed9.png">

Reply:
Bad name 💯 haha
The only tricky thing is thinking of something that makes sense and differs from the workflow title, considering that this workflow only has one step.